test/uk/gov/hmrc/plasticpackagingtax/registration/views/group/ContactDetailsNameViewSpec.scala

```diff
@@ -121,7 +162,48 @@ class ContactDetailsNameViewSpec extends UnitViewSpec with Matchers {
 
       view must haveGovukGlobalErrorSummary
 
-      view must haveGovukFieldError("lastName", "Enter the main contact’s last name")
+      view must haveGovukFieldError("lastName", "Enter the last name of the main contact")
+    }
+
+    "last name is spaces" in {
+
+      val form = MemberName
+        .form()
+        .fillAndValidate(MemberName("first", "   "))
+      val view = createView(form)
+
+      view must haveGovukGlobalErrorSummary
+
+      view must haveGovukFieldError("lastName", "Enter a last name")
+    }
+
+    "last name is invalid" in {
+
+      val form = MemberName
+        .form()
+        .fillAndValidate(MemberName("first", "1224"))
+      val view = createView(form)
+
+      view must haveGovukGlobalErrorSummary
+
+      view must haveGovukFieldError(
+        "lastName",
+        "Last name must only include letters, hyphens, spaces, apostrophes and full stops"
+      )
+    }
+
+    "last name is > 35" in {
+
+      val form = MemberName
+        .form()
+        .fillAndValidate(
+          MemberName("first", "hjghjgfhjdsgfhjdsgfdshjfgdsjhfgjshfgdshjfgdsjhfgdsjhfgdsjh")
```

Comment:
this is difficult to count
```suggestion
          MemberName("first", "x" * 35)
```
or should that be 34